app/client/components/Client.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import SignUp from './sign-up/SignUp';
+import SignIn from './sign-in/SignIn';
+import EnsureUserLoggedIn from '../components/EnsureLoggedInContainer';
+import HomePageRedirect from '../components/HomepageRedirectContainer';
+import PageNotFound from '../components/PageNotFound';
+import ResetPassword from '../components/reset-password/EnterEmailForm';
+import NewPassword from '../components/reset-password/NewPasswordForm';
+
+if (module.hot) {
+  module.hot.accept();
+}
+
+/**
+ * Overarching Client Component
+ * @returns {ReactRouting} Definition of app routes
+*/
+function Client() {
```

Comment:
Please refer to your Javascript Style guide on how to create React components, also try and get a proper eslint to lint your project accordingly